packages/wp-story-editor/src/components/postLock/postLock.js

```diff
@@ -160,6 +167,27 @@ function PostLock() {
     return () => clearInterval(timeout);
   }, [postLockInterval, currentUserLoaded]);
 
+  useEffect(() => {
+    if (
+      enablePostLockingTakeOver &&
+      improvedAutosaves &&
+      showLockedDialog &&
+      user &&
+      hasNewChanges &&
+      !isFirstTime
```

Comment:
Curious why only when `!isFirstTime`?

Reply:
The take over message should show until this is not the first time.

Reply:
To Miina's point, the variable wording is not the most intuitive, but at this point I guess it is what it is :-)